base/markdown/render/plain.jl

```diff
@@ -20,9 +20,12 @@ function plain{l}(io::IO, header::Header{l})
 end
 
 function plain(io::IO, code::Code)
-    println(io, "```", code.language)
+    # If the code includes a fenced block this will break parsing,
+    # so it must be enclosed by a longer ````-sequence.
+    n = mapreduce(length, max, matchall(r"`+", code.code)) + 1
```

Comment:
Should the regex only match backticks that begin at the start of a line, ie ``r"^`+"m``?

Also, if `code.code` contains no matches `mapreduce` would throw

```
julia> mapreduce(length, max, matchall(r"`+", ""))
ERROR: ArgumentError: reducing over an empty collection is not allowed
 in _mapreduce at ./reduce.jl:139
 in mapreduce at ./reduce.jl:159
 in mapreduce at ./reduce.jl:162
 in eval at ./boot.jl:265
```

since `matchall` returns a 0-length vector. Is that a valid edge case?